Sources/AblyAssetTrackingSubscriber/WorkerQueue/SubscriberWorkerQueueProperties.swift

```diff
@@ -1,7 +1,181 @@
 import Foundation
 import AblyAssetTrackingInternal
 
-struct SubscriberWorkerQueueProperties: WorkerQueueProperties
-{
+struct SubscriberWorkerQueueProperties: WorkerQueueProperties {
     public var isStopped = false
+    var presenceData: PresenceData
+    let ablySubscriber: Subscriber
```

Comment:
I don't like this bit tbh - the way I implemented it is we pass the ablySubscriber instance here so that we can set it as a sender when firing delegate methods, for example:
```
delegate?.subscriber(sender: ablySubscriber, didUpdatePublisherPresence: publisherPresence)
```
The main reason for that was to avoid changing the api by changing the `sender` type, but it complicates the rest of the code quite a bit, having to introduce this weird `configureWorkerQueue` method on default subscriber... So maybe in this case changing the api a bit would be justified, and the `sender` in delegate methods should just be of type `SubscriberWorkerQueueProperties `. Thoughts?

Reply:
Good suggestions, thanks! 👍  846237855e2d882a68495a0d63106d3d3939e815